### PR TITLE
Add macro guards for variables, which used/not used depending on BE-type

### PR DESCRIPTION
**Pull request by evgeniusp**
Fix some issues after the fixing of warnings: add macro guards for variables, which used/not used depending on BE-type to avoid wanings during compilation.

Signed-off-by: Pavlov, Evgeniy <evgeniy.pavlov@intel.com>

### CODE REVIEW

**Review by capatober — DISMISSED** [review]
Looks correct to me

**Review by rarutyun — APPROVED** [review]
I think it looks good. I give my approve but if time allows I would like to know the reason before merging

**Review by capatober — APPROVED** [review]

**Pull request merged**
